src/inventory.cc

```diff
@@ -4755,6 +4756,31 @@ static int _barter_attempt_transaction(Object* a1, Object* a2, Object* a3, Objec
     return 0;
 }
 
+static int _barter_get_quantity_moved_items(
+    Object* item,
+    int maxQuantity,
+    bool fromPlayer,
+    bool fromInventory
+) {
+    if (maxQuantity <= 1) {
+        return maxQuantity;
+    }
+
+    int suggestedValue = 1;
+    if (item->pid == PROTO_ID_MONEY && !gGameDialogSpeakerIsPartyMember) {
+        // Calculate change money automatically
+        int totalCostPlayer = objectGetCost(_ptable);
+        int totalCostNpc = _barter_compute_value(gDude, _target_stack[0]);
+        // Actor's balance: negative - the actor must add money to balance the tables and vice versa
+        int balance = fromPlayer ? totalCostPlayer - totalCostNpc : totalCostNpc - totalCostPlayer;
+
+        if ( (balance < 0 && fromInventory) || (balance > 0 && !fromInventory) ) {
```

Comment:
```suggestion
        if ((balance < 0 && fromInventory) || (balance > 0 && !fromInventory)) {
```
Eliminates Clang warning.